### PR TITLE
[LUMI-28] Add minimal info on env vars

### PR DESCRIPTION
**Pull request by javiermtorres**
This PR lists and adds a short description to the environment variables expected by the startup scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,18 @@ pants package lumigator/python/mzai/backend --no-local-cache
...
+### Environment variable reference
+
+If the S3 storage service is used, some environment variables are needed to connect to it. Additionally, models from Mistral or OpenAI can be used via API instead of instantiating them within Lumigator. In this case, the corresponding key is needed.
+
+| Environment variable name | Default value | Description |
+| --- | :-: | --- |
+| LOCAL_FSSPEC_S3_ENDPOINT_URL | "" | Endpoint URL for the S3 data storage service |
+| LOCAL_FSSPEC_S3_KEY | "" | Key for the S3 data storage service |
+| LOCAL_FSSPEC_S3_SECRET | "" | Secret for the S3 data storage service |
+| MISTRAL_API_KEY | "" | Key for Mistral API models |
```
[the remark this answers is not in the document]
Comment (veekaybee) [reply README.md:143]: We'll want to cross-reference the env vars to the local documentation as well, in specifying how to set up the local app https://github.com/mozilla-ai/lumigator/tree/main/.devcontainer#running-lumigator-locally-with-docker-compose

[the remark this answers is not in the document]
Comment (javiermtorres) [reply README.md:143]: > Should we include MISTRAL and OPENAI here

I can split it in two tables. Using different headings would be overkill imho right now, though, so they'll just have their own small paragraph like "these are the storage vars".

[the remark this answers is not in the document]
Comment (javiermtorres) [reply README.md:143]: > We'll want to cross-reference the env vars to the local documentation

@veekaybee At least a ref, sure. Would you like me to include small snippets like, "the Mistral API needs to be defined as per the (config)[ref]" too, in the appropriate points?

[the remark this answers is not in the document]
Comment (veekaybee) [reply README.md:143]: Yup, perfect

**Review by veekaybee — APPROVED** [review]
Thanks for adding these!

**Pull request merged**